### DISCUSSION TITLE
opkg installed but not found

### DISCUSSION THREAD

**Discussion by Manfred73** [Entware/Entware#769] [Q&A] [answered]
I installed Entware via qnapstore and it's available under /share/ZFS530_DATA/.qpkg/Entware.
Also /opt/bin is present although I don't see it added to the path (also not after exit shell and relogin).
I've also did a manual install using wget and run qpkg package manually from command line. Same result.
When I go to /opt/bin I can see the opkg binary and when I run it from there, it still says command not found.
Doing sh opkg from there it says "Cannot execute binary file".
How can I install/use opkg. I'd like to use it in order to install tools such as sqlite3.
uname -a gives me "Linux QNAP 5.10.60-qnap #1 SMP Wed Nov 17 06:28:43 CST 2021 x86_64 GNU/Linux", so if I understand correctly, I should be able to run opkg?

**Comment by zyxmon** [comment]
1. Make sure your are using `admin` user. Entware PATH is added only for admin
2. If it does not work for `admin`, post the installation log (`sh qpkg_file`)
3. There is a lot of posts and solutions on qnap forum - https://forum.qnap.com/viewforum.php?f=351

**Reply by Manfred73** [reply]
I did switch to admin. Where do I find the installation log file?

**Reply by Manfred73** [reply]
And just reading that su -i won't work?

**Comment by zyxmon** [comment]
>> Where do I find the installation log file?

console log , not a file
>> And just reading that su -i won't work?

su from firmware works (never used it), there are problems with su from entware.

**Reply by Manfred73** [reply]
I installed it both through qnapstore and manually after sudo to admin. Both didn't work.
I can't use admin account directly since it's disabled.

**Comment by zyxmon** [comment]
>> I can't use admin account directly since it's disabled.

In this case - modify Entware sturtup script - few lines after ` # adding opkg apps into system path ...` change /root/.profile to your new  superuser.

**Comment by micjgam** [comment]
Followed the OP on a Synology 920+ and getting:

     -ash: opkg: command not found

@Entware/opt is where it should be.

**Comment by zyxmon** [comment]
Entware developers do not have Synology. Please ask synology users on synology forum for help. I strongly recommend you to publish installation log. I guess there are some network problems to access entware repo during installation.

**Comment by micjgam** [comment, marked as answer]
Got it...

I had set task user to my user admin account rather than root.  It works now.

Bit of a noob here, but what is the additional comments regarding iperf3?
